src/Raven.Client/Documents/Operations/Replication/UpdatePullReplicationAsSinkOperation.cs

```diff
@@ -24,9 +24,9 @@ public UpdatePullReplicationAsSinkOperation(PullReplicationAsSink pullReplicatio
             if (pullReplication.CertificateWithPrivateKey != null)
             {
                 var certBytes = Convert.FromBase64String(pullReplication.CertificateWithPrivateKey);
-                using (var certificate = new X509Certificate2(certBytes,
+                using (var certificate = CertificateLoaderUtil.CreateCertificate(certBytes,
                     pullReplication.CertificatePassword,
-                    X509KeyStorageFlags.Exportable | X509KeyStorageFlags.MachineKeySet))
+                    X509KeyStorageFlags.Exportable))
```

Comment:
better to use CertificateLoaderUtil.FlagsForExport

Reply:
done